PendingReleaseNotes.md

```diff
@@ -37,3 +37,4 @@ v1.6...
 * Monitors failover can be [disabled](Documentation/ceph-mon-health.md#failing-over-a-monitor), this is useful for monitor going under planned maintenance where automatic failover is not desired
 * RGW: Rook has started using deployment's replicatset functionality instead of deploying multiple deployments for each rgw
 * Support running Volume Replication Controller on the RBD provisioner pod. Depends on Cephcsi v3.3.0
+* Disable Host Network in CSI CephFS and RBD nodeplugin pods by default. `CSI_ENABLE_HOST_NETWORK` in `rook-ceph-operator-config` configmap and `enableCSIHostNetwork` in helm values.yaml can be set to true to enable Host Network
```

Comment:
We don't need that anymore since the 1.6 is out, I just reset the file here https://github.com/rook/rook/pull/7692. This will be included in the 1.6.1 note but not in the `PendingReleaseNotes.md` file.